src/tools/kata-ctl/src/utils.rs

```diff
@@ -5,7 +5,10 @@
 
 #![allow(dead_code)]
 
-use anyhow::{anyhow, Result};
+use anyhow::{anyhow, Context, Result};
+use std::fs;
+use std::io::Write;
+use tempfile::tempdir;
```

Comment:
To overcome the static errors reported you should remove this line: `use tempfile::tempdir;`

Reply:
@dborquez I have moved the use statement under the test module where it is required. Similarly for the `use std::io::Write;` it is required by tests.